### ISSUE TITLE
`W191` triggers on tabs in multi-line strings

### ISSUE THREAD

**Issue by stinodego** [charliermarsh/ruff#3438]
The following file gets flagged. 

```python
greeting = """hello
	world
"""
```

The lint should not trigger here, as this is not Python program indentation, it's simply part of a string. It should be no different from the following example, which is NOT flagged:

```python
greeting = """hello
world	:)
"""
```

This is also a known issue in pycodestyle:
https://github.com/PyCQA/pycodestyle/issues/376

**Comment by charliermarsh** [comment]
Ah yeah, kind of a "known" bug which was deemed correct because of pycodestyle's behavior. If it's considered a bug in pycodestyle, though, we can definitely fix.

**Comment by charliermarsh** [comment]
Closed by #3837.
